scripts/tests/test_suites.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+find_result=$(find src/app/tests/suites -type f -name "*.yaml" -exec basename {} \; | awk -F '.yaml' '{print $1}' | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/;/g')
```

Comment:
The `sed` command here is inscrutable enough that it might be worth adding a comment explaining what this is doing, exactly.  Building a `;`-separated list of test names, but what does the `'$!ba'` have to do with that, or the `':a'`, etc.